model/facade/action/SyncCalendarAction.php

```diff
@@ -57,9 +57,9 @@ protected function doExecute()
 
         $client->setCalendar($arrayOfCalendars[ConfigurationParametersManager::getParameter('CALENDAR_ID')]);
 
-        $lastYear = date("Y") - 1;
         $nextYear = date("Y") + 1;
-        $start = gmdate("Ymd\THis\Z", strtotime($lastYear . "-01-01"));
+
+        $start = gmdate("Ymd\THis\Z", date(strtotime("-3 months")));
         $end = gmdate("Ymd\THis\Z", strtotime($nextYear . "-12-31"));
         $currentEvents = $client->getEvents($start, $end);
```

Comment:
I think it's better to rename these variables now that we are using the start inside the loop to avoid it being overwritten in the line 73.

```suggestion
        $startSyncPeriod = gmdate("Ymd\THis\Z", date(strtotime("-3 months")));
        $endSyncPeriod = gmdate("Ymd\THis\Z", strtotime($nextYear . "-12-31"));
        $currentEvents = $client->getEvents($startSyncPeriod, $endSyncPeriod);
```